x-pack/plugins/security_solution/server/lib/source_status/elasticsearch_adapter.ts

```diff
@@ -8,6 +8,7 @@ import { FrameworkAdapter, FrameworkRequest } from '../framework';
 import { SourceStatusAdapter } from './index';
 import { buildQuery } from './query.dsl';
 import { ApmServiceNameAgg } from './types';
+import { ENDPOINT_METADATA_INDEX } from '../../../common/constants';
```

Comment:
this should work, but we'll also wanna make sure that the endpoint events and alerts index are added to the defaults.  Can be in another PR